tests/telemetry/conftest.py

```diff
@@ -84,6 +84,12 @@ def mock_telemetry_client(mock_requests_get, mock_requests):
     client._clean_up()
 
 
+@pytest.fixture(autouse=True)
+def is_mlflow_testing(monkeypatch):
+    # enable telemetry by default when running tests in local with dev version
+    monkeypatch.setattr(mlflow.telemetry.utils, "_IS_MLFLOW_TESTING", True)
```

Comment:
```suggestion
    monkeypatch.setattr(mlflow.telemetry.utils, "_IS_MLFLOW_TESTING_TELEMETRY", True)
```

can we rename this? `_IS_MLFLOW_TESTING` is confusing.

Reply:
Yep